ai/app/models/model_def.py

```
@@ -0,0 +1,100 @@
+MODEL_PATH = os.path.abspath(os.path.dirname(__file__))
+TOKENIZER_NAME = "hun3359/klue-bert-base-sentiment"
+MAX_TOKEN = 512
+THRESHOLD = 0.54  # 최적 threshold
+
+# 토크나이저와 모델 로드
+tokenizer = AutoTokenizer.from_pretrained(TOKENIZER_NAME)
+config = AutoConfig.from_pretrained(MODEL_PATH)
+model = AutoModelForSequenceClassification.from_pretrained(MODEL_PATH, config=config)
+model.eval()
+
+def chunk_text_with_kss(text, tokenizer, max_tokens=512):
```

Pylint (3.3.7)

[refactor] Too many local variables (16/15)

(R0914)